### PR TITLE
Fix compilation of Qdatastream (windows, visual 2013)

### PR DESCRIPTION
**Pull request by slurdge**
This is a simple compilation fix. Application would not compile with Qt5, Visual 2013 on windows

### CODE REVIEW

**Comment by smarx** [comment]
_Automated message from Dropbox CLA bot_

@slurdge, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/) and update the thread so we can consider merging your code.

File: src/cookiejar.cpp [hunk 1]
```diff
@@ -35,6 +35,7 @@
 #include <QTimer>
 #include <QDebug>
 #include <QNetworkCookie>
+#include <Qdatastream>
```
Comment (lfranchi) [inline src/cookiejar.cpp]: I think you want to capitalize this as `QDataStream`

**Comment by lfranchi** [comment]
Thanks for the contribution! Once you've signed the CLA / fixed the capitalizations I'll be happy to merge.

**Comment by slurdge** [comment]
Hi, I signed the CLA and fixed the capitalizations!

**Comment by lfranchi** [comment]
Thanks!

**Pull request merged**